js/components/accessibilityControls.js

```diff
@@ -23,8 +23,17 @@ AccessibilityControls.prototype = {
     var currentTime;
     var newPlayheadTime;
     var newVolume;
+    var targetTagName;
 
-    if (e.keyCode === CONSTANTS.KEYCODES.SPACE_KEY){
+    if (e.target && typeof e.target.tagName === "string") {
```

Comment:
This `typeof` is mostly just checking for the existence of `tagName`. It really shouldn't be anything other than a string, but I'm checking it this way just to make sure it doesn't crash the `toLowerCase` below if there's a weird browser bug.